Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -0,0 +1,1098 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Display;
+using Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Helpers;
+using Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Parse;
+using Windows.UI;
+using Windows.UI.Core;
+using Windows.UI.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Documents;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// An efficient and extensible control that can parse and render markdown.
+    /// </summary>
+    public sealed partial class MarkdownTextBlock : Control, ILinkRegister
+    {
+        /// <summary>
+        /// The default color used for backgrounds of sub elements.
+        /// </summary>
+        private static Color _defaultSubElementBgColor = Color.FromArgb(255, 224, 224, 224);
```

Comment:
Because you use them as default value for CodeBackGround for instance I would just set CodeBackground to this value in XAML and that's it :)